test/unit/specs/components/staking/PageValidator.spec.js

```diff
@@ -207,8 +207,22 @@ describe(`PageValidator`, () => {
       ).not.toBeNull()
     })
 
-    it("shows formatted validator website url", () => {
-      expect(wrapper.vm.$el).toMatchSnapshot()
+    it(`shows empty website url`, () => {
+      validator.description.website = ``
+      wrapper.setData({ validator })
+      expect(wrapper.vm.website).toBe(`--`)
+    })
+
+    it(`shows https website url`, () => {
+      validator.description.website = `www.monty.ca`
+      wrapper.setData({ validator })
+      expect(wrapper.vm.website).toBe(`https://www.monty.ca`)
+    })
+
+    it(`shows http website url`, () => {
```

Comment:
i would add one more test that checks to make sure nothing happens if it already has `https`

Reply:
Good call. Thanks for the suggestion.